site/docs/relations/logical_relations.md

```diff
@@ -136,8 +154,9 @@ The join operation will combine two separate inputs into a single output, based
 | Outer | Return all records from both the left and right inputs. For each cross input match, return a record including the data from both sides. For any remaining non-match records, return the record from the corresponding input along with nulls for the opposite input. |
 | Left  | Return all records from the left input. For each cross input match, return a record including the data from both sides. For any remaining non-matching records from the left input, return the left record along with nulls for the right input. |
 | Right | Return all records from the right input. For each cross input match, return a record including the data from both sides. For any remaining non-matching records from the right input, return the left record along with nulls for the right input. |
-
-
+| Semi | Returns records from the left input. These are returned only if the records have a join partner on the right side. |
+| Anti  | Return records from the left input. These are returned only if the records do not have a join partner on the right side. |
+| Single | Returns at most one join partner per entry on the left input. In case no partner is found, NULL is returned. |
```

Comment:
Can you add more insight into what happens if there is more than one partner (e.g. operation can use arbitrarily among partners). Also, can you clarify the null statement. I assume you always return the left side as in a left outer join. I believe you are saying here that null is returned for the right fields when no partner is identified.

Reply:
Hey Jacques, I've updated the text a bit.

Just as an extra clarification: 
The single join is defined in this paper http://btw2017.informatik.uni-stuttgart.de/slidesandpapers/F1-10-37/paper_web.pdf
In section 3.2.

One thing that might differ from system to system, is the execution on a single join when more than a join partner is found per tuple.

DuckDB follows the SQLite semantics for subqueries, which means that in the case of >1 result, we just return the first result we find, rather than throwing an error. But as you can see in sec 3.2, an error can be thrown as well. Hence I've also decided to add it there.